### PR TITLE
PD-Fund.-W2

### PR DESCRIPTION
**Pull request by esma-g**
## What does this change?

Module: Fundamentals
Week(s): 2

## Checklist

- [x] I have read the [contributing guidelines](https://syllabus.codeyourfuture.io/contributing/overview)
- [x] I have checked my spelling and grammar with an [automated tool](https://www.grammarly.com/grammar-check)
- [x] I have previewed my changes to check the [markdown renders](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax) as I intend
- [x] I have run my code to check it works
- [x] My changes follow our [Style Guide](https://syllabus.codeyourfuture.io/guides/code-style-guide)

## Description

- Learning objectives deleted
- Coursework duration added
- Exercise markup format fixed
- Session and exercise objectives added

## Rendered Pages
https://syllabus.codeyourfuture.io/fundamentals/week-2/session
https://personal-development.codeyourfuture.io/em-modules/pd-session-4/week-2-feedback

### CODE REVIEW

**Review by kfklein15 — DISMISSED** [review]
lgtm

File: docs/fundamentals/week-2/learning-objectives.md [hunk 1]
```diff
@@ -29,10 +29,6 @@ sidebar_label: Learning Objectives
 
 ## PD
 
-- [ ] Explain the purpose of Personal Development
-- [ ] Believe in own capabilities and ability to be successful
-- [ ] Understand own strengths
-- [ ] Identify areas of development
 - [ ] Communicate challenges and barriers in public
 - [ ] Recognise and address fears of transparency
 - [ ] Take active initiative to overcome barriers to success
```
Comment (SallyMcGrath) [inline docs/fundamentals/week-2/learning-objectives.md]:
```suggestion
- [ ] Take active initiative to overcome barriers to success
```

Still a bit confused by this one. Can you explain more?

**Review by SallyMcGrath — DISMISSED** [review]
A question on this but it is non blocking

File: docs/fundamentals/week-2/learning-objectives.md (hunk 1 above)
Comment (esma-g) [reply docs/fundamentals/week-2/learning-objectives.md]: The PD learning objectives are taken from [the PD competencies rubric](https://docs.google.com/spreadsheets/d/1S0h66lDGXwc1RiuufJDzB5R-YFrr2qNMIq5xxai_hxA/edit#gid=2116060541). I was not involved in their production. I'm trying to match them to the weekly content while moving from the PD website into the Syllabus website.

**Comment by esma-g** [comment]
Deleting PD learning objectives and not adding any new ones until they are re-created by the PD syllabus review

**Review by kfklein15 — APPROVED** [review]
Ok

**Review by illicitonion — APPROVED** [review]

**Pull request merged**